app/src/main/java/org/p2p/wallet/common/analytics/TrackerContract.kt

```diff
@@ -1,18 +1,14 @@
 package org.p2p.wallet.common.analytics
 
-interface TrackerContract {
-
-    fun logEvent(event: String, params: Array<out Pair<String, Any>>? = null) = Unit
-
-    fun setUserProperty(key: String, value: String) = Unit
-
-    fun incrementUserProperty(property: String, byValue: Int = 1) = Unit
+import org.p2p.wallet.utils.NoOp
 
-    fun setUserPropertyOnce(key: String, value: String) = Unit
-
-    fun setUserId(userId: String?) = Unit
-
-    fun appendToArray(property: String, value: Int) = Unit
-
-    fun regenerateDeviceId() = Unit
+interface TrackerContract {
+    fun logEvent(event: String, params: Map<String, Any> = emptyMap()) = NoOp
+    fun logEvent(event: String, params: Array<out Pair<String, Any>>) = NoOp
+    fun setUserProperty(key: String, value: String) = NoOp
+    fun incrementUserProperty(property: String, byValue: Int = 1) = NoOp
+    fun setUserPropertyOnce(key: String, value: String) = NoOp
+    fun setUserId(userId: String?) = NoOp
+    fun appendToArray(property: String, value: Int) = NoOp
+    fun regenerateDeviceId() = NoOp
```

Comment:
These looks very weird. 
Let's leave as `fun someName()` without any type? 
I thought we are added `NoOp` for functions realization

Reply:
no need to ask me, it was just `= Unit` before me. i'll remove NoOp